### PR TITLE
LPD-18577 Create ScriptManagmentConfiguration component and GroovyScriptUses Modal

### PR DESCRIPTION
**Pull request by danvb112**
Related Issue: https://liferay.atlassian.net/browse/LPD-18577

### CODE REVIEW

**Comment by danvb112** [comment]
Hey @igor-franca, this PR is blocked because it depends of https://github.com/liferay-objects/liferay-portal/pull/3169

**Comment by danvb112** [comment]
ci:test:stable

**Comment by danvb112** [comment]
ci:test:sf

**Comment by danvb112** [comment]
ci:test:sf

**Comment by danvb112** [comment]
ci:test:sf

File: modules/apps/portal-security/portal-security-script-management-web/jest-setup.config.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
```
Comment (igor-franca) [inline modules/apps/portal-security/portal-security-script-management-web/jest-setup.config.js:2]: Hey @danvb112, use `2024` here 😆

File: modules/apps/portal-security/portal-security-script-management-web/src/main/resources/META-INF/resources/js/tests/GroovyScriptUsesModalContent.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,98 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
```
Comment (igor-franca) [inline modules/apps/portal-security/portal-security-script-management-web/src/main/resources/META-INF/resources/js/tests/GroovyScriptUsesModalContent.spec.tsx:2]: Hey @danvb112, use 2024 here 😆

File: modules/apps/portal-security/portal-security-script-management-web/src/main/resources/META-INF/resources/js/tests/ScriptManagementContainer.spec.tsx [hunk 3]
```diff
@@ -0,0 +1,98 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
```
Comment (igor-franca) [inline modules/apps/portal-security/portal-security-script-management-web/src/main/resources/META-INF/resources/js/tests/ScriptManagementContainer.spec.tsx:2]: Hey @danvb112, use 2024 here 😆

**Review by igor-franca — CHANGES_REQUESTED** [review]
Hey @danvb112, good work. I couldn't find significant points of improvement, just one detail.

**Comment by danvb112** [comment]
ci:test:sf

**Review by igor-franca — APPROVED** [review]

**Review by rebsilva — APPROVED** [review]
LGTM! :tada: only this error is happening when i try to save but i think it's because the BE is not ready yet right? @danvb112 
Screenshot from 2024-03-21 18-45-34

**Comment by danvb112** [comment]
Yes! Exactly @rebsilva!

**Comment by danvb112** [comment]
ci:forward:force

**Comment by danvb112** [comment]
ci:forward:force

**Pull request closed without merging**